src/titiler/xarray/titiler/xarray/main.py

```diff
@@ -0,0 +1,30 @@
+"""titiler xarray app."""
+
+from fastapi import FastAPI
+
+from titiler.xarray.factory import TilerFactory
+
+###############################################################################
+
+app = FastAPI(
+    openapi_url="/api",
+    docs_url="/api.html",
+    description="""Xarray based tiles server for MultiDimensional dataset (Zarr/NetCDF).
+
+---
+
+**Documentation**: <a href="https://developmentseed.org/titiler/" target="_blank">https://developmentseed.org/titiler/</a>
+
+**Source Code**: <a href="https://github.com/developmentseed/titiler" target="_blank">https://github.com/developmentseed/titiler</a>
+
+---
+    """,
+)
+
+md = TilerFactory(router_prefix="/md")
+
+app.include_router(
+    md.router,
+    prefix="/md",
+    tags=["Multi Dimensional"],
+)
```

Comment:
TODO: remove this before merging 

`titiler.xarray` should be seen as a plug-in to titiler and not as an application itself. We will add example on how to build application using the endpoint factory